app/uk/gov/hmrc/cataloguefrontend/serviceconfigs/ServiceConfigsService.scala

```diff
@@ -352,6 +352,10 @@ object ServiceConfigsService {
     def hasDecommissioningDownstream: Boolean =
```

Comment:
can we remove this now?

Reply:
I opted to keep in the banner for decomissioned downstream services (using this def). I was thinking though users should have made sure that no downstream services are using this before they get to the decomission step it may not stop them from disregarding the instructions. Thought keeping this in may help that just in case rather than possibly losing sight of these services.

Reply:
It's easily added back if we want to do this. And dead-code will just be deleted by someone else (unless intention is commented).

I expect if we wanted to do this, we'd just extend `hasDeprecatedDownstream` to cover both states for simplicity (Note Decommissioning status is really only intended for the service owners to do their job and is expected to be a short-lived status - Deprecation status is the one to advise for clients).

I think our efforts would be best spent on highlighting archived/deleted downstream dependencies if we end up in this state. It could encourage config clean up if nothing else.